cmd/rookflex/cmd/expander.go

```diff
@@ -0,0 +1,68 @@
+package cmd
```

Comment:
The other flex methods in the `cmd` pkg don't have unit tests because of the complication with the rpc calls. If we can get an integration test for resize I think it will be sufficient.